src/countryInfo.js

```diff
@@ -0,0 +1,68 @@
+import { getAllApiData } from "./app";
+
+dataFilter()
+async function dataFilter() {
+
+    //filters data
+
+    let data = await getAllApiData()
+    let list = []
+
+    for (let item of data) {
```

Comment:
Hier kun je spreading toepassen, zodat je nier overal `item` vóór hoeft te zetten
```suggestion
    for (let { name, flag, population, region } of data) {
```